overview/intro.md

```diff
@@ -2,67 +2,113 @@
+If a publisher would prefer to implement their own solution, [source code](https://github.com/prebid/prebid-server) is available from our Github page and detailed instructions for configuring, deploying and testing your implementation can be found in the [Prebid Server section](/prebid-server/prebid-server-overview.html) of thi site.  
 
-![Ad Ops Diagram]({{ site.github.url }}/assets/images/adops-intro.png){: .pb-lg-img :}
+**Prebid Server process**
 
-## Further Reading
+Prebid Server provides multiple endpoints for auctions as well as data retrieval and supports the AMP (accelerated mobile pages) format. The primary endpoint is `/openrtb2/auction` and the process follows these steps: 
 
+1. An OpenRTB request is sent to Prebid Server. 
+2. Prebid Server conducts auction with selected SSPs and ad exchanges. 
+3. PBS collects the bids and creatives and stores them in Prebid cache. 
+4. PBS passes the winning bid parameters to the ad server. 
+5. The ad server takes the passed in key-values and finds a matching line item and then compares to  other line items that bid on this impression. 
+6. If the ad server determines Prebid wins the auction it returns a reference to the winning creative to Prebid. 
+7. Prebid retrieves the winning creative and displays it in the correct ad slot. 
+
+![Prebid Server Flowchart](/assets/images/flowcharts/pb-server-display.png) 
+
+### Prebid Mobile
+
+For mobile apps, Prebid provides Prebid Mobile (PBM), an end-to-end header bidding solution for both iOS and Android. Working in conjunction with Prebid Server, PBM reduces latency and enables access to more mobile buyers, 
+PBM provides options for banner and interstitial ad formats and enables user to set global targeting values for the bid request. 
+
+**Prebid Mobile process**
+
+The PBM header bidding process follows these steps: 
+
+1. Prebid Mobile sends a request to Prebid Server. This request consists of the Prebid Server account ID and config ID for each tag included in the request.
+2. Prebid Server constructs an OpenRTB bid request and passes it to the demand partners.
+3. Each demand partner returns a bid response to Prebid Server. The bid response includes the bid price and the creative content.
+4. Prebid Server sends the bid responses to Prebid Mobile.
+5. Prebid Mobile sets key-value targeting for each ad slot through the primary ad server mobile SDK. This targeting will activate one or more of Prebid line items that were previously configured in the primary ad server.
+6. If the line item associated with the Prebid Mobile bid wins, the primary ad server returns the Prebid Mobile creative JavaScript to the ad server’s SDK.
+7. The Prebid Mobile creative JavaScript will fetch and render the corresponding creative con *<----This is from the existing mobile overview, it just stops at con...I am going to follow up with Punnaghai to correct this*
```

Comment:
I looked at the history, it originally said this: 
The Prebid Mobile creative JavaScript will fetch and render the corresponding creative content from the winning Prebid Server demand partner.

Reply:
Thanks, fixed.